frameworks/PHP/symfony/src/Controller/BenchController.php

```diff
@@ -91,20 +93,19 @@ public function updateRawAction(Request $request)
       $worlds = array();
       $conn = $this->get('database_connection');
 
-      for($i = 0; $i < $queries; ++$i) {
+      for ($i = 0; $i < $queries; ++$i) {
           $id = mt_rand(1, 10000);
-          $random_number = mt_rand(1, 10000);
-          $conn->executeUpdate('UPDATE World SET randomNumber=? WHERE id=?', array($random_number, $id));
-          $worlds[] = array('id' => $id, 'randomNumber' => $random_number);
+          $randomNumber = mt_rand(1, 10000);
+          $conn->executeUpdate('UPDATE world SET randomNumber=? WHERE id=?', array($randomNumber, $id));
```

Comment:
wouldn't it be better/faster to prepare outside of the loop, then execute?

Reply:
I will check if there I can notice a sensible difference

Reply:
I tried this way, but it's a bit slower (+5ms in tests 50 users):
```php
    $stmt = $conn->prepare('SELECT * FROM world WHERE id = ?');

    for ($i = 0; $i < $queries; ++$i) {
        $stmt->execute(array(mt_rand(1, 10000)));
        $worlds[] = $stmt->fetch();
    }
```